tsconfig.test.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "target": "es5",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+    "module": "esnext",
```

Comment:
Can we avoid of using Nextjs?

Reply:
In general yes. However this line is related to `esnext`, which is a JS standard, not a package related to next.